src/components/Book.js

```diff
@@ -1,20 +1,28 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
+import { removeBook } from '../redux/books/booksSlice';
 import CircularProgress from './CircularProgressbar';
 
 const Book = ({ book }) => {
   const {
-    category, title, author, chapter,
+    itemId, title, author, category,
   } = book;
+  // const {
+  //   id, category, title, author, chapter,
+  // } = book;
 
   Book.propTypes = {
     book: PropTypes.node.isRequired,
+    // id: PropTypes.string.isRequired,
     category: PropTypes.string.isRequired,
     title: PropTypes.string.isRequired,
     author: PropTypes.string.isRequired,
-    chapter: PropTypes.string.isRequired,
+    // chapter: PropTypes.string.isRequired,
```

Comment:
- [ ] It is unprofessional to leave commented codes in your codebase so i kindly suggest you remove code that are not necessary for the working of your application so that your codebase would be inline with clean code practices :+1: :sparkles: